packages/labs/react/src/create-component.ts

```diff
@@ -4,6 +4,47 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
+// Typecast that curries an Event type through a string to match
```

Comment:
Can we remove the "curries" part here? It's not consistent with my understanding of the definition of the term.

Reply:
sounds good, curry is farther than where i started, makes sense

Reply:
done!